client/src/components/previews/LocationPreview.js

```diff
@@ -54,14 +58,37 @@ const LocationPreview = ({ className, uuid }) => {
         <PreviewField label="Name" value={location.name} />
 
         <PreviewField
-          label="Status"
-          value={Location.humanNameOfStatus(location.status)}
+          label="Type"
+          value={Location.humanNameOfType(location.type)}
         />
 
         <PreviewField
-          label="Type"
-          value={Location.humanNameOfType(location.type)}
+          label="Latitude, Longitude"
```

Comment:
This is using an incorrect label, see GeoLocation for the correct label (depends on dictionary setting).